apps/full-site-editing/full-site-editing-plugin/starter-page-templates/page-template-modal/index.js

```diff
@@ -24,6 +36,8 @@ import ensureAssets from './utils/ensure-assets';
 /* eslint-enable import/no-extraneous-dependencies */
 
 const DEFAULT_HOMEPAGE_TEMPLATE = 'maywood';
+const INSERTING_HOOK_NAME = 'isInsertingPageTemplate';
+const INSERTING_HOOK_NAMESPACE = 'automattic/full-site-editing/inserting-template';
```

Comment:
Should we define these in a separate file so they can be imported in `tracking.js`?

Reply:
I made the same suggestion at the beginning, but in some way, it's like in WordPress filters, right? You need to know the exact name of the filter in order to get rightly hooked.
Anyway, I agree with adding a global file definition, too.

Reply:
Since FSE and wpcom-block-editor are completely separate apps and I believe there is no precedent of any shared code other than npm-based libraries, I think we should keep them separate. In WordPress backend you also write all the filter names directly in files and you don't use a central place with a list of filters to import.